crates/symbolicator-js/src/symbolication.rs

```diff
@@ -105,21 +105,29 @@ async fn symbolicate_js_frame(
     // we check for a valid line (i.e. >= 1) first, as we want to avoid resolving / scraping the minified
     // file in that case. we frequently saw 0 line/col values in combination with non-js files,
     // and we want to avoid scraping a bunch of html files in that case.
-    let line = if raw_frame.lineno > 0 {
-        raw_frame.lineno
-    } else {
-        return Err(JsModuleErrorKind::InvalidLocation {
-            line: raw_frame.lineno,
-            col: raw_frame.colno,
-        });
+    let line = match raw_frame.lineno {
+        Some(lineno) if lineno > 0 => lineno,
+        lineno => {
+            return Err(JsModuleErrorKind::InvalidLocation {
+                line: lineno.unwrap_or(0),
+                col: raw_frame.colno,
+            });
+        }
+    };
+
+    let abs_path = match raw_frame.abs_path {
+        Some(ref abs_path) => abs_path,
+        None => {
+            return Err(JsModuleErrorKind::InvalidAbsPath);
+        }
     };
```

Comment:
Just a rust nit:

```suggestion
    let Some(abs_path) = &raw_frame.abs_path else {
        return Err(JsModuleErrorKind::InvalidAbsPath);
    };
```

Reply:
Both approaches appear to fail compilation. I'm going to revert this change for now.
```
error[E0502]: cannot borrow `*raw_frame` as mutable because it is also borrowed as immutable
   --> crates/symbolicator-js/src/symbolication.rs:145:17
    |
105 |   ...ath = raw_frame
    |  __________-
106 | | ...path
    | |_______- immutable borrow occurs here
...
145 |   ...   apply_source_context(raw_frame, minified_source.contents(...
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ mutable borrow occurs here
...
163 |   ...ap_or_else(|| abs_path.to_owned());
    |                    -------- immutable borrow later captured here by closure

```